### PR TITLE
Remove redundant entries from dictionary.

### PR DESCRIPTION
**Pull request by Talljoe**
### About

Removes some redundant outlines from the dictionary. These entries all are identical to their individual components. See [this gist](https://gist.github.com/Talljoe/f620e83a5cd78789452c569758dee377).

### CODE REVIEW

File: plover/assets/main.json [hunk 1]
```diff
@@ -1213,7 +1204,6 @@
 "A*D/HOBG": "ad hoc",
 "A*D/HOEBG": "ad hoc",
 "A*D/HR*EUB": "ad lib.",
-"A*D/HREUB": "ad lib",
```
Comment (morinted) [inline plover/assets/main.json]: Let's leave this one in because it might be looked up.

File: plover/assets/main.json [hunk 2]
```diff
@@ -6097,7 +6070,6 @@
 "AEUR/K-PB/*ER": "air conditioner",
 "AEUR/K-PB/ER": "air conditioner",
 "AEUR/K-PB/ERS": "air conditioners",
-"AEUR/K-PBG": "air conditioning",
```
Comment (morinted) [inline plover/assets/main.json]: We can leave this one

File: plover/assets/main.json [hunk 3]
```diff
@@ -12661,13 +12611,9 @@
 "AUGS/*PLT/*EUPB": "Augmentin",
 "AUGT": "ought",
 "AUL": "all",
-"AUL/-F/AEU/SUD": "all of a sudden",
 "AUL/-F/EU/SUD": "all of a sudden",
```
Comment (morinted) [inline plover/assets/main.json]: Let's remove this misstroke entry

File: plover/assets/main.json [hunk 4]
```diff
@@ -35910,7 +35778,6 @@
 "KHEFT": "chest",
 "KHEFT/*ER": "Chester",
 "KHEFT/-F/TKRAUR/-S": "chest of drawers",
```
Comment (morinted) [inline plover/assets/main.json]: remove the -S version too

File: plover/assets/main.json [hunk 5]
```diff
@@ -62645,7 +62414,6 @@
 "PHRAPB/*BG": "plank",
 "PHRAPB/*BG/-S": "planks",
 "PHRAPB/*ER": "planner",
-"PHRAPB/-BG": "plan being",
 "PHRAPB/-BG/TOPB": "plankton",
```
Comment (morinted) [inline plover/assets/main.json]: Let's also remove plan/k/ton and plan/ks

File: plover/assets/main.json [hunk 6]
```diff
@@ -64840,7 +64604,6 @@
 "PORPB/TKPWRAEF/EUBG": "pornographic",
 "PORPB/TKPWRAEFBG": "pornographic",
 "PORPB/TKPWRAF/EUBG": "pornographic",
-"PORPB/TKPWRAFBG": "important graphic",
```
Comment (morinted) [inline plover/assets/main.json]: Can we add `PORPB/TKPWRA*FBG` as "pornographic"?

File: plover/assets/main.json [hunk 7]
```diff
@@ -86219,7 +85898,6 @@
 "SAPB/TOR/SKWRO": "Santoro",
 "SAPB/TOS": "Santos",
 "SAPB/TPO*RD": "Sanford",
-"SAPB/TPRAPB": "San Fran",
```
Comment (morinted) [inline plover/assets/main.json]: I'd leave this one in for searchability

File: plover/assets/main.json [hunk 8]
```diff
@@ -95265,7 +94918,6 @@
 "SKWROPB/THA*PB": "Jonathan",
 "SKWROPB/THAPB": "Jonathan",
 "SKWROPB/THAPB/SWEUFT": "Jonathan Swift",
-"SKWROPB/TKAO*ER": "John Deere",
```
Comment (morinted) [inline plover/assets/main.json]: This can be left in

File: plover/assets/main.json [hunk 9]
```diff
@@ -95756,8 +95406,6 @@
 "SO*UT/KRAOE/KWRA": "South Korea",
 "SO*UT/KWREPL/-PB": "South Yemen",
 "SO*UT/PARBG": "South Park",
-"SO*UT/PHERBG": "South America",
```
Comment (morinted) [inline plover/assets/main.json]: Let's leave in South America/n, and South Wales

File: plover/assets/main.json [hunk 10]
```diff
@@ -105239,7 +104840,6 @@
 "STR*ULG": "struggle",
 "STR*ULGD": "struggled",
 "STR*UPBG": "Strunk",
-"STR*UPBG/SKP/WHAO*EUT": "Strunk and White",
 "STR*UPBG/SKP/WHAOEUT": "Strunk and White",
```
Comment (morinted) [inline plover/assets/main.json]: Let's remove both Strunk and White entries

File: plover/assets/main.json [hunk 11]
```diff
@@ -123206,7 +122720,6 @@
 "TKRAOEUF/PW*EU": "drive-by",
 "TKRAOEUF/SHAFT": "driveshaft",
 "TKRAOEUF/TEUF": "derivative",
-"TKRAOEUF/WAEU": "drive way",
 "TKRAOEUF/WAEUS": "driveways",
```
Comment (morinted) [inline plover/assets/main.json]: Let's remove plural driveways

File: plover/assets/main.json [hunk 12]
```diff
@@ -131069,9 +130532,6 @@
 "TPHO*RT/KRAOE/KWRA": "North Korea",
 "TPHO*RT/KRAOE/KWRAPB": "North Korean",
 "TPHO*RT/KWREPL/-PB": "North Yemen",
-"TPHO*RT/KWRORBG": "North York",
```
Comment (morinted) [inline plover/assets/main.json]: I'd keep these three too

File: plover/assets/main.json [hunk 13]
```diff
@@ -134274,7 +133696,6 @@
 "TPRA*PBG/HREU": "frankly",
 "TPRA*PBG/HREUPB": "Franklin",
 "TPRA*PBG/HREUPB/TKEL/TPHOE": "Franklin Delano",
-"TPRA*PBG/HROEUD/WRAO*EUT": "Frank Lloyd Wright",
```
Comment (morinted) [inline plover/assets/main.json]: Leave this name

File: plover/assets/main.json (hunk 6 above)
Comment (Talljoe) [inline plover/assets/main.json]: Pornographic already has the outline `PORPB/TKPWRAEFBG`, do you still want the one with `*`?"

**Comment by morinted** [comment]
Yeah, I think it makes sense to have it.

File: plover/assets/main.json (hunk 3 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 4 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 5 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 6 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 7 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 10 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 11 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 8 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 12 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 13 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 9 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 2 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

File: plover/assets/main.json (hunk 1 above)
Comment (Talljoe) [inline plover/assets/main.json]: Done.

**Comment by morinted** [comment]
Thank you very much.

It was interesting to see the most of those entries that were probably caused by Mirabai trying to override bad entries that were word boundaries.

**Pull request merged**